.circleci/config.yml

```diff
@@ -37,8 +37,13 @@ jobs:
             docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
             docker buildx create --name multiarch --driver docker-container --use
             docker buildx inspect --bootstrap
+            CORE_VERSION_WITH_PATCH=$(cat Dockerfile | grep "ARG CORE_VERSION=" | cut -d'=' -f2)
             CORE_VERSION=$(cat Dockerfile | grep "ARG CORE_VERSION=" | cut -d'=' -f2 | cut -d'.' -f1,2)
-            docker buildx build --platform $BUILDX_PLATFORMS -t supertokens/supertokens-postgresql:${CORE_VERSION} -o type=image,push=true .
+            if [ "$CORE_VERSION" != "$CORE_VERSION_WITH_PATCH" ]; then
```

Comment:
no need have this if condition. Here, CORE_VERSION will always be X.Y and CORE_VERSION_WITH_PATCH will always be X.Y.Z. Which means the condition `if [ "$CORE_VERSION" != "$CORE_VERSION_WITH_PATCH" ]` can never be false.

Reply:
I wrote this condition only for the case when n docker file we ignore patch, in that case these will be same. Assuming that never happens, ignoring it for now.